Add profile badge and badge-weighted seeding articles (#7534)
ppy/osu-wiki: wiki/Community/Profile_badge/en.md

---
tags:
  - badges
  - profile badges
  - user badge
  - user badges
---

# Profile badge

*Not to be confused with [group](/wiki/Community/Group) badges, which also appear on user profiles.*\
*For rules regarding usage as tournament prizing, see: [Official tournament support § Profile Badges](/wiki/Tournaments/Official_support#profile-badges)*

![Screenshot of HappyStick's badges on the website](img/HappyStick.png "Profile badges display under the avatar and username on the website.")

> **Profile badges** (commonly referred to as just *badges*) are small graphics on user profile pages awarded for various achievements. They are most commonly given as [tournament](/wiki/Tournaments) and [contest](/wiki/Contests) prizes, but have other uses, such as for [community contributors](/wiki/People/Community_Contributors), [Beatmap Spotlights](/wiki/Beatmap_Spotlights#rewards), and continued [group](/wiki/Community/Group) membership.

Author: they aren't really same thing though. "User" prefix is because it's in relation to a user, but I'm not planning to write "Group" article like that

> **Profile badges** (commonly referred to as just *badges*) are small graphics on user profile pages awarded for various achievements. They are most commonly given as [tournament](/wiki/Tournaments) and [contest](/wiki/Contests) prizes, but have other uses, such as for [community contributors](/wiki/People/Community_Contributors), [Beatmap Spotlights](/wiki/Beatmap_Spotlights#rewards), and continued [group](/wiki/Community/Group) membership.

Reviewer: ok

> When hovering over a badge, a tooltip is shown with more details about why the badge was awarded. Badges can also be linked to other web pages for more context, but none currently use this feature.

Author: y tho

> When hovering over a badge, a tooltip is shown with more details about why the badge was awarded. Badges can also be linked to other web pages for more context, but none currently use this feature.

Author: moved to trivia instead, assuming that's along the lines of why u commented..?

> When hovering over a badge, a tooltip is shown with more details about why the badge was awarded. Badges can also be linked to other web pages for more context, but none currently use this feature.

Reviewer: sure, definitely fits better there. i meant that it's pretty unnecessary information if it's a thing that potentially could exist but doesn't

## Tournament seeding

*Main page: [Badge-weighted seeding](/wiki/Tournaments/Badge-weighted_seeding)*

> In the [game client](/wiki/Client) and on the website, badges are purely decorative. However, due to [tournament](/wiki/Tournaments) prize badges showcasing players' performance, some tournaments have adopted seeding methods that factor in the amount of badges players have, commonly known as [badge-weighted seeding](/wiki/Tournaments/Badge-weighted_seeding) (BWS).

Reviewer: ```suggestion
In the [game client](/wiki/Client) and on the website, badges are purely decorative. However, due to [tournament](/wiki/Tournaments) prize badges showcasing players' performance, some tournaments have adopted [seeding](https://en.wikipedia.org/wiki/Seed_(sports)) methods that factor in the amount of badges players have, commonly known as [badge-weighted seeding](/wiki/Tournaments/Badge-weighted_seeding) (BWS).
```

## Old website

![Screenshot of Uan's badges on the old website](img/Uan.png "On the old website, profile badges stacked under the username and title.")

Rather than stacking horizontally in a container spanning the whole page, the old website stacked badges vertically between a user's username and country flag.

> A side-effect of this layout scheme was that the default, non-expanded height of [userpages](/wiki/osu!supporter#editable-profile-section) could be extended by means of earning more badges. While unintended, it became a running gag in the tournament community.<!-- More example userpages beyond Toy would be nice, else "running gag" doesn't have a source. -clayton --> ![][flag_US] [Toy](https://osu.ppy.sh/users/2757689) showcased this in [a popular Tweet](https://twitter.com/droombs/status/1036050610687074304), showing off his record for most badges won at the time of posting.<!-- I couldn't find a definitive source for this but I'm fairly certain that was part of the joke. Toy was the first to reach 10 badges about a year before this, and he was tied with DigitalHypno at 20 a few months later. Please add a source as a reference if you can find one! -clayton -->

Reviewer: also this is literally trivia

```suggestion
A side effect of this layout scheme was that the default, non-expanded height of [userpages](/wiki/osu!supporter#editable-profile-section) could be extended by means of earning more badges. While unintended, it became a running gag in the tournament community.<!-- More example userpages beyond Toy would be nice, else "running gag" doesn't have a source. -clayton --> ![][flag_US] [Toy](https://osu.ppy.sh/users/2757689) showcased this in [a popular Tweet](https://twitter.com/droombs/status/1036050610687074304), showing off his record for most badges won at the time of posting.<!-- I couldn't find a definitive source for this but I'm fairly certain that was part of the joke. Toy was the first to reach 10 badges about a year before this, and he was tied with DigitalHypno at 20 a few months later. Please add a source as a reference if you can find one! -clayton -->
```

Author: i think it's fine in this section cuz it's kinda de-facto "history" already, talking about the old website, and this was actually pretty well-known rather than some niche technical thing.

**Profile badges** (commonly referred to as just *badges*) are small graphics on user profile pages awarded for various achievements. They are most commonly given as [tournament](/wiki/Tournaments) and [contest](/wiki/Contests) prizes, but have other uses, such as rewards for [community contributors](/wiki/People/Community_Contributors), [Beatmap Spotlights](/wiki/Beatmap_Spotlights#rewards), and continued [group](/wiki/Community/Group) membership.

When hovering over a badge, a tooltip is shown with more details about why the badge was awarded.

## Tournament seeding

*Main page: [Badge-weighted seeding](/wiki/Tournaments/Badge-weighted_seeding)*

In the [game client](/wiki/Client) and on the website, badges are purely decorative. However, due to [tournament](/wiki/Tournaments) prize badges showcasing players' performance, some tournaments have adopted [seeding](https://en.wikipedia.org/wiki/Seed_(sports)) methods that factor in the amount of badges players have, commonly known as [badge-weighted seeding](/wiki/Tournaments/Badge-weighted_seeding) (BWS).

> ## Old website

Reviewer: ```suggestion
## History
```

Author: i feel like this section title implies it would be more comprehensive, but it's actually just a quick explanation of the look on the new-old website and random userpage height fact

Reviewer 2: I think I prefer history since it will likely be expanded in the future if/as more changes are made.

![Screenshot of Uan's badges on the old website](img/Uan.png "On the old website, profile badges stacked under the username and title.")

Rather than stacking horizontally in a container spanning the whole page, the old website stacked badges vertically between a user's username and country flag.

A side effect of this layout scheme was that the default, non-expanded height of [userpages](/wiki/osu!supporter#editable-profile-section) could be extended by means of earning more badges. While unintended, it became a running gag in the tournament community.<!-- More example userpages beyond Toy would be nice, else "running gag" doesn't have a source. -clayton --> ![][flag_US] [Toy](https://osu.ppy.sh/users/2757689) showcased this in [a popular Tweet](https://twitter.com/droombs/status/1036050610687074304), showing off his record for most badges won at the time of posting.<!-- I couldn't find a definitive source for this but I'm fairly certain that was part of the joke. Toy was the first to reach 10 badges about a year before this, and he was tied with DigitalHypno at 20 a few months later. Please add a source as a reference if you can find one! -clayton -->

## Trivia

> - The first two profile badges were awarded to ![][flag_PL] [niedzwiedz1124](https://osu.ppy.sh/users/9610) and ![][flag_PL] [White Wolf](https://osu.ppy.sh/users/39828) on 6 September, 2009, for [winning *Tag Tournament*](https://osu.ppy.sh/community/forums/topics/17169).<!-- TODO: The changelog entry for badges being added is also on the same day, and should be mentioned. are these available online anywhere to link to? -clayton -->

Reviewer: for the badge, you could embed it as an image by linking to its asset.ppy.sh link

https://assets.ppy.sh/profile-badges/tagwinner.png

> - Badges can be linked to other web pages for more context, but none currently use this feature.

Reviewer: ```suggestion
- Badges can link to other web pages for more context, but none currently use this feature.
```

![Screenshot of Uan's badges on the old website](img/Uan.png "On the old website, profile badges stacked under the username and title.")

Rather than stacking horizontally in a container spanning the whole page, the old website stacked badges vertically between a user's username and country flag.

A side effect of this layout scheme was that the default, non-expanded height of [userpages](/wiki/osu!supporter#editable-profile-section) could be extended by means of earning more badges. While unintended, it became a running gag in the tournament community.<!-- More example userpages beyond Toy would be nice, else "running gag" doesn't have a source. -clayton --> ![][flag_US] [Toy](https://osu.ppy.sh/users/2757689) showcased this in [a popular Tweet](https://twitter.com/droombs/status/1036050610687074304), showing off his record for most badges won at the time of posting.<!-- I couldn't find a definitive source for this but I'm fairly certain that was part of the joke. Toy was the first to reach 10 badges about a year before this, and he was tied with DigitalHypno at 20 a few months later. Please add a source as a reference if you can find one! -clayton -->

## Trivia

::: Infobox

> ![](https://assets.ppy.sh/profile-badges/tagwinner.png "Tag Tournament first place badge")

Reviewer: ```suggestion
![](https://assets.ppy.sh/profile-badges/tagwinner.png "Tag Tournament first-place badge")
```
